tests/libnet/hotplug.go

```diff
@@ -38,7 +38,12 @@ import (
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
 )
 
-func VerifyDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, queueCount int32) *v1.VirtualMachineInstance {
+func VerifyDynamicInterfaceChange(
+	vmi *v1.VirtualMachineInstance,
+	queueCount int32,
+	timeout,
+	pollInterval time.Duration,
```

Comment:
I do not think this is needed, but it is not worth investing into this too much either.

I mainly do not like it because it exposes the implementation of polling and makes it too noisy.

Reply:
I would appreciate an alternative suggestion.